web_generator/lib/src/js/helpers.dart

```diff
@@ -0,0 +1,140 @@
+import 'dart:collection';
+import 'dart:js_interop';
+
```

Comment:
Let's leave a comment/TODO with the bug you filed to add these types to `dart:js_interop` to remove these once that's done.

(super cool to see some of this implementation btw)